back/app/policies/phase_policy.rb

```diff
@@ -38,4 +38,10 @@ def survey_results?
   def submission_count?
     survey_results?
   end
+
+  def delete_inputs?
+    return unless active?
```

Comment:
A method ending with a `?` should return a boolean.

```suggestion
    return false unless active?
```